src/components/login-form.jsx

```diff
@@ -0,0 +1,85 @@
+import { Form, Formik } from "formik";
+import { Button, Col, Container, Row } from "react-bootstrap";
+import * as yup from "yup";
+import { useDispatch } from "react-redux";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import { save } from "../features/user/user-slice";
+import userService from "../services/api/user-service";
+import LoginFormTextField from "./login-form-text-field";
+
+const schema = yup.object({
+    email: yup.string().email("Invalid email address").required("Required"),
+    password: yup.string().required("Required"),
+});
+
+export default function LoginForm() {
+    const dispatch = useDispatch();
+
+    const handleLogin = (values) =>
+        userService.login(values).then((response) => dispatch(save(response)));
```

Comment:
To sto je ovo sve bez zagrada mi se malo ne svidja, mada su i u nasem react boilerplate-u tako radili. Ovo sve ispadne kao neki oneline-er 😂
```
const handleLogin = (values) => (
    userService.login(values).then((response) =>
        dispatch(save(response))
    );
);
```
Posto handeLogin funkcija i ima zapravo tu jednu liniju samo, dovoljno je koristiti obicne (male) zagrade, ne moraju `{}`.

Reply:
Stavio sam `{}` posto mi `()` skloni Prettier ekstenzija za formatiranje :P

Reply:
A voli prettier da smara, moze da se podesi sve to ali neka, moze i `{}`